app/views/devise/sessions/new.html.erb

```diff
@@ -1,14 +1,8 @@
-<div class="fluid-container center floor signin">
 <h2 class="new-edit-title">Sign in</h2>
 
 <% if flash[:error] %>
   <div class="alert alert-danger" role="alert"><%= flash[:error] %></div>
 <% end %>
-
-<%- if devise_mapping.omniauthable? %>
-  <%- resource_class.omniauth_providers.each do |provider| %>
-    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), class: "button", id:"back", method: :post, data: { 'turbo-method' => :post } %><br />
-    <%= button_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), class: "button", method: :post %><br />
-  <% end -%>
+<%- Devise.omniauth_providers.each do |provider| %>
+  <%= button_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(OmbuLabs::Auth.user_class, provider), method: :post %><br />
```

Comment:
button_to in the default view, app's may not have rails-ujs, can be overriden with a new view to use link_to